src/scripts/utils.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+from constants import _constants
+
+def _write_file(settings, content):
+    file_count = 1
+    output_dir = os.path.abspath(settings.output_dir)
+    test_case_count = 0
+    if not os.path.isdir(output_dir):
+        os.makedirs(output_dir)
+    while True:
+        filename = settings.generated_filename.format(NUMBER = file_count)
+        file_to_open = os.path.join(output_dir, filename)
+        with open(file_to_open, "w") as file:
+            file.write("%s\n" % (settings.file_header))
+            counter = settings.test_case_in_a_file
+            for i in range(test_case_count, len(content)):
+                file.write("%s" %(content[i]))
+                counter -= 1
+                test_case_count += 1
```

Comment:
`test_case_count` is increases while `counter` decreases in each iteration. How about merging these values?

Reply:
Done